cisagov/XFD
# Alphabetize state selection in RegisterForm.tsx

#  #

## 🗣 Description ##

Front-end:
The registration form contains a field for selecting US States and Territories. This dropdown is not alphabetized and should do so. This code will be consolidated in a "constants" file for both the organization form and registration form to use. Issue #19 




## 💭 Motivation and context ##

To improve user experience, we need to sort the states alphabetically.





## 🧪 Testing ##

- The US States and Territories dropdown should display options in alphabetical order.
- The sorting should be case-insensitive (e.g., “New York” should come after “North Carolina”).
- The change should be reflected in both the organization form and the registration form.
- Create a centralized function or module that handles state-related logic.
- This function/module should populate the state, stateName, and regionId fields consistently for both User and Organization entities.
- Ensure that the consolidated logic is efficient and maintainable.







## ✅ Pre-approval checklist ##






      in code comments.
      to reflect the changes in this PR.

## ✅ Pre-merge checklist ##






## ✅ Post-merge checklist ##
